src/makefile

```diff
@@ -19,12 +19,11 @@ CC = nvcc
 ifdef OS # For Windows with the cl.exe compiler
 CFLAGS = -O3 -arch=sm_60 -Xcompiler "/wd 4819"
 else # For linux
-CFLAGS = -std=c++14 -O3 -arch=sm_60
+CFLAGS = -std=c++14 -O3 -arch=sm_75 -DUSE_NETCDF
 endif
-INC = -I./
```

Comment:
@erhart1 Are these changes, using NETCDF and links to `home/repos/netcdf`, suppose to be in the makefile?

Reply:
Sorry accidental inclusion. Will fix it.